cluster-stamp.json

```diff
@@ -848,160 +860,12 @@
                 "description": "All azure advisor alerts"
```

Comment:
investigate whether we can move this to shared resources stamp and enroll resources later (ideal) or remove the scope and keep this at the subs lvl (second best). Otherwise make them unique.

Reply:
Done | Addressed from 01518b87fcb4f2a51cab216612de5fab63362167

cc/ @ckittel I had to go with the third option. I know it wast the least desired 😞 but based on [docs](https://docs.microsoft.com/en-us/azure/templates/microsoft.insights/activitylogalerts#ActivityLogAlert) I would say is our only option. 

![image](https://user-images.githubusercontent.com/158487/109305469-845c7000-781c-11eb-95bf-11e6edeac299.png)

Btw it seems you could dup the name without issues, please take a look at the screenshots below for two different rg:

![image](https://user-images.githubusercontent.com/158487/109305378-65f67480-781c-11eb-8293-3bc471876cd5.png)

![image](https://user-images.githubusercontent.com/158487/109305337-55de9500-781c-11eb-9d2d-bb5532a438b9.png)

Reply:
hold on with this. Just found an example in another object giving the scope the subscription id while requires "resource ids". Just testing it to see if it works.

Reply:
it did work, and it now addressed from [96373](https://github.com/mspnp/aks-secure-baseline/pull/158/commits/96373535c9f271774b7434489d692ac30ae998ae)